lib/posticle/Cargo.toml

```diff
@@ -1,13 +1,16 @@
 [package]
 name = "posticle"
 description = "tools for working with Twitter-post-like text"
-version = "0.1.0"
+version = "0.2.0"
 authors = ["Erin Moon <erin@hashbang.sh>"]
```

Comment:
feel free to add your name here, you basically rewrote the lib

Reply:
Cool!